scientific_library/tvb/simulator/coupling.py

```diff
@@ -501,3 +501,6 @@ def pre(self, x_i, x_j):
 
     def post(self, gx):
         return self.a / gx.shape[0] * gx
+
+
+CouplingWithCurrentState = [Difference, Kuramoto]
```

Comment:
undocumenated global `list` that anyone could modify during runtime, please use an `enum` or something more appropriate

Reply:
The variable is not used and it can be remove.